packages/datetime/src/common/classes.ts

```diff
@@ -34,6 +34,7 @@ export const DATEPICKER_DAY_IS_TODAY = `${DATEPICKER_DAY}--isToday`;
 export const DATEPICKER_DAY_WRAPPER = `${DATEPICKER}-day-wrapper`;
 export const DATEPICKER_FOOTER = `${DATEPICKER}-footer`;
 export const DATEPICKER_MONTH_SELECT = `${DATEPICKER}-month-select`;
+export const DATEPICKER_TODAY_BUTTON_DISABLED = `${NS}-disabled`;
```

Comment:
no need for this class, it's unused. also it's just an alias for the disabled class from the core package

```suggestion
```